### PR TITLE
JAMES-2401 Adding occupation when getting user quotas

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java [hunk 1]
```diff
@@ -79,7 +86,13 @@ void setUp() throws Exception {
         usersRepository.addUser(BOB.asString(), PASSWORD);
         MailboxSessionMapperFactory factory = null;
         userQuotaRootResolver = new DefaultUserQuotaRootResolver(factory);
-        UserQuotaService userQuotaService = new UserQuotaService(maxQuotaManager, userQuotaRootResolver);
+
+        currentQuotaManager = mock(CurrentQuotaManager.class);
+        Mockito.when(currentQuotaManager.getCurrentMessageCount(any())).thenReturn(QuotaCount.count(0));
+        Mockito.when(currentQuotaManager.getCurrentStorage(any())).thenReturn(QuotaSize.size(0));
```
Comment (rouazana) [inline server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java]: static import

File: server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java [hunk 2]
```diff
@@ -421,6 +434,54 @@ public void getQuotaShouldReturnBothWhenValueSpecified() throws Exception {
         softly.assertThat(jsonPath.getLong("global." + COUNT)).isEqualTo(22);
     }
 
+    @Test
+    public void getQuotaShouldReturnOccupation() throws Exception {
+        maxQuotaManager.setMaxStorage(userQuotaRootResolver.forUser(BOB), QuotaSize.size(80));
+        maxQuotaManager.setMaxMessage(userQuotaRootResolver.forUser(BOB), QuotaCount.count(100));
+        Mockito.when(currentQuotaManager.getCurrentStorage(any())).thenReturn(QuotaSize.size(40));
+        Mockito.when(currentQuotaManager.getCurrentMessageCount(any())).thenReturn(QuotaCount.count(20));
```
Comment (rouazana) [inline server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java]: here you override what's set it setup?

**Comment by rouazana** [comment]
did you miss to update the documentation?

File: server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java (hunk 1 above)
Comment (chibenwa) [reply server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java]: It's clashing with **RestAssured::when**

File: server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java (hunk 2 above)
Comment (chibenwa) [reply server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java]: Yes.  It's set up to return nothing. And in these tests I want something...

Comment (rouazana) [reply server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java]: ok in these cases I thought we did not define the value in the setup to be clear about the intent

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**